File: UST  JAVA/Factorial.java
class Factorial
{
	public static void main(String[] args)
	{
	   double result= factorialOfnumber(5);
	   System.out.println(	result);
	}
	static double factorialOfnumber( int num)
	{
		int fact =1;
 		for (int i =num ; i >=1; i--)
		{
			fact = fact*i;
			
		}
		return fact;
	}
}

File: JAVA Script/Assignment/assign2/factorial.js
console.log('==with function name==')
function factorial(num)
 {
    var factorial = 1,i;
    for(i=1; i<=num;i++)
    factorial = factorial*i;
    console.log('fact=', factorial);
}
factorial(5)

console.log('===with function expression')
var factorial = function(num){
    var factorial = 1,i;
    for(i=1; i<=num;i++)
    factorial = factorial*i;
    console.log('fact=', factorial);
}
var fact = factorial(5)
console.log('==self invoked function==');
(function (num)
 {
    var factorial = 1,i;
    for(i=1; i<=num;i++)
    factorial = factorial*i;
    console.log('fact=', factorial);
}) (5)
 console.log('==with fat arrow function==');
 var factorial = (num)=>{
    var factorial = 1,i;
    for(i=1; i<=num;i++)
    factorial = factorial*i;
    console.log('fact=', factorial);
}
 factorial(5)